### PR TITLE
add security aspect in intro + small type fix

### PR DESCRIPTION
**Pull request by sebastiankb**
also see #904 and #925

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1278.html" title="Last updated on Nov 22, 2021, 3:07 PM UTC (e5a33e5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1278/815fc8e...e5a33e5.html" title="Last updated on Nov 22, 2021, 3:07 PM UTC (e5a33e5)">Diff</a>

### CODE REVIEW

File: index.template.html [hunk 1]
```diff
@@ -405,13 +405,13 @@ <h2>Thing Description</h2>
...
-  (much like the <i>index.html</i> of a Web site). A TD instance has four main
+  (much like the <i>index.html</i> of a Web site). A TD instance has five main
   components: textual metadata about the <a href="#thing">Thing</a> itself,
   a set of <a href="#interactionaffordance">Interaction Affordances</a>
   that indicate how the <a>Thing</a> can be used,
   <a href="#sec-data-schema-vocabulary-definition">schemas</a> for the data
-  exchanged with the <a>Thing</a> for machine-understandability,
-  and, finally, <a href="#sec-hypermedia-vocabulary-definition">Web links</a> to 
+  exchanged with the <a>Thing</a> for machine-understandability, <a href="#sec-security-vocabulary-definition">Security Definitions</a>
+  provide metadata about the security mode of the <a>Thing</a> that must be used for secure interaction, and, finally, <a href="#sec-hypermedia-vocabulary-definition">Web links</a> to
```
Comment (egekorkan) [inline index.template.html:414]: to provide 

the previous sentences have this kind of sentence structure

Comment (egekorkan) [reply index.template.html:414]: Another point:
about the security mechanisms that must be used for interactions

secure interaction is a bit weird for me but maybe just me

**Review by egekorkan — CHANGES_REQUESTED** [review]

File: index.template.html (hunk 1 above)
Comment (sebastiankb) [reply index.template.html]: done

**Review by egekorkan — APPROVED** [review]

**Comment by sebastiankb** [comment]
from today's TD, need to check where the commit for #925 is

**Comment by sebastiankb** [comment]
based on last TD call, going to merge. #925 is now covered

**Pull request merged**